numpy/core/tests/test_shape_base.py

```diff
@@ -157,10 +157,6 @@ def test_2D_array(self):
     def test_generator(self):
         with assert_warns(FutureWarning):
             hstack((np.arange(3) for _ in range(2)))
-        if sys.version_info.major > 2:
-            # map returns a list on Python 2
-            with assert_warns(FutureWarning):
-                hstack(map(lambda x: x, np.ones((3, 2))))
```

Comment:
Is there a reason you removed this test?

Reply:
I accidentally reverted it, added back, tested locally and pushed.